Sources/SwiftProtobufCore/Message+BinaryAdditions.swift

```diff
@@ -97,7 +72,7 @@ extension Message {
   ///   - options: The BinaryDecodingOptions to use.
   /// - Throws: `BinaryDecodingError` if decoding fails.
   @inlinable
-  public init<Bytes: ContiguousBytes>(
+  public init<Bytes: SwiftProtobufContiguousBytes>(
```

Comment:
The existing `ContiguousBytes` apis sorta broke the model, but it used to be the apis used "Serialized" for all the "binary" encoding based apis (and "JSON" and "TextFormat" for those).

Since we're dropping all the "Data" apis that are named with "Serialized", should we do a slight api correct and call this: `init(serializedBytes:...)`

@tbkka ?

Reply:
I'm fine with making that change, I actually prefer the name myself.

Reply:
I like that idea, too.